### DISCUSSION TITLE
Documentation query: backing up config

### DISCUSSION THREAD

**Discussion by mn113** [cupcakearmy/autorestic#293] [Q&A] [answered]
This (highlighted) piece of the documentation seems ambiguous to me:

<img width="920" alt="Screenshot 2023-03-08 at 11 51 07" src="https://user-images.githubusercontent.com/3638829/223693956-977c82d3-3945-4b1a-b38c-a9d34a3fa118.png">

Is it saying 1. "be sure to back up the config _before_ the first run, because it's better not to have the encryption keys in the backup",
or 2. "backing it up now is fine, or at any point from here on is also fine"
or 3. If the encryption keys are pretty important not to lose, why doesn't it recommend to run the first backup and _then_ back up the modified config?

**Comment by cupcakearmy** [comment, marked as answer]
You are right. It should be after, as you need the encryption keys. 3 is the way to go.
